colossus-tests/src/test/scala/colossus/controller/PipeSpec.scala

```diff
@@ -84,7 +84,7 @@ class PipeSpec extends WordSpec with MustMatchers {
       pipe.push(1) must equal(Success(PushResult.Ok))
       val r = pipe.push(1) 
       var triggered = false
-      r mustBe an[Success[PushResult.Full]]
+      r mustBe an[Success[_]]
```

Comment:
These tests needs to be further fixed, simply checking if its a Success is not enough